model_prices_and_context_window.json

```diff
@@ -2749,6 +2749,24 @@
         "litellm_provider": "ollama",
         "mode": "completion"
     },
+    "ollama/llama3": {
+        "max_tokens": 8192,
+        "max_input_tokens": 8192,
+        "max_output_tokens": 8192,
+        "input_cost_per_token": 0.00000010,
+        "output_cost_per_token": 0.00000010,
```

Comment:
does ollama have a hosted endpoint ? why is there a cost here

Reply:
Good catch, an oversight during copy&paste - I'll fix that.